internal/utils/models/store.go

```diff
@@ -43,18 +43,11 @@ type TStrategyFromRiskRiskScores struct {
 	TeamKnowledgeScore  float64 `json:"teamKnowledgeScore"`
 	TestingScore        float64 `json:"testingScore"`
 }
-type TStrategyFromRiskRiskProfile struct {
-	High   float64 `json:"high"`
-	Low    float64 `json:"low"`
-	Median float64 `json:"median"`
-}
 type TStrategyFromRisk struct {
-	RiskScores  TStrategyFromRiskRiskScores  `json:"riskScores"`
-	RiskProfile TStrategyFromRiskRiskProfile `json:"riskProfile"`
-	Tokens      []string                     `json:"tokens"`
+	RiskScores TStrategyFromRiskRiskScores `json:"riskScores"`
```

Comment:
Good point, but could we leave this as is so that we could add more metrics back in?
Ideally we should have additional risk profile scores and recommended allocations of tvl as well, but I would like to first safely migrate this risks json file